src/query/service/src/pipelines/processors/transforms/transform_sort_spill.rs

```diff
@@ -435,3 +478,165 @@ impl Merger {
         Ok(Some(block))
     }
 }
+
+pub fn create_transform_sort_spill(
```

Comment:
It's better to make the `create_transform_sort_spill` as a function of the spiller not a outside function, make it more object easy to maintain.

Reply:
We cannot put `create_transform_sort_spill` into the struct because `TransformSortSpill` has generics.